### ISSUE TITLE
DeprecationWarning with Python 3.6

### ISSUE THREAD

**Issue by Anteru** [pygments/pygments-migration-test#1018]
(Original issue 1314 created by The-Compiler on 2016-12-29T17:18:38.653599+00:00)

When importing pygments with a Python 3.6 started with `-Wall`, I get:

```
.../site-packages/pygments/util.py:27: DeprecationWarning: Flags not at the start of the expression <(.+?)(\s.*?)?>.*?</ (truncated)
  tag_re = re.compile(r'<(.+?)(\s.*?)?>.*?</.+?>(?uism)')
```

**Comment by Anteru** [comment]
(Original issue 1314 last updated on 2017-01-18T06:37:34.830356+00:00)

(Issue automaticaly closed due to status in Bitbucket: resolved)

**Comment by Anteru** [comment]
(Original comment by birkenfeld on 2017-01-18T06:35:38.711493+00:00)

Interesting. There's no good reason for the flags to be in the back, so I'll fix this.

**Comment by Anteru** [comment]
(Original comment by birkenfeld on 2017-01-18T06:37:34.842360+00:00)

Fixed by pull request #687.
